test/repl/proxysql_repl_tests/bin/proxy-repl-casc-config.bash

```
@@ -7,18 +7,18 @@ if [ -z "${INFRA}" ]; then
     echo "INFRA is empty, please check env vars passed to: " ${0}
     exit 1
 elif [ -z "${RHG2}" ]; then
-    echo "RHG2 is empty, please check env vars passed to: " ${0}
+    echo "RHG1 is empty, please check env vars passed to: " ${0}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)